### PR TITLE
Testsuite: replace wget by curl

### PR DESCRIPTION
**Pull request by jordimassaguerpla**
## What does this PR change?

wget is not in leap micro or sle micro

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [X] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

File: testsuite/features/step_definitions/navigation_steps.rb [hunk 1]
```diff
@@ -1047,7 +1047,7 @@
     end
   # debian based systems don't come with curl installed
   if (os_family.include? 'debian') || (os_family.include? 'ubuntu')
-    node.run_until_ok("wget --no-check-certificate -qO- #{protocol}://#{system_name}:#{port}#{path} | grep -i '#{text}'")
+    node.run_until_ok("curl --insecure -s #{protocol}://#{system_name}:#{port}#{path} | grep -i '#{text}'")
   else
     node.run_until_ok("curl -s -k #{protocol}://#{system_name}:#{port}#{path} | grep -i '#{text}'")
```
Comment (vandabarata) [inline testsuite/features/step_definitions/navigation_steps.rb:1052]: can't we just remove the if condition in here and use what's in the else for this and the other cases?

Comment (vandabarata) [reply testsuite/features/step_definitions/navigation_steps.rb:1052]: wait, but actually, if debian based systems don't come with wget installed, can we even run this there?

Comment (srbarrios) [reply testsuite/features/step_definitions/navigation_steps.rb:1052]: Yup, Vanda is right, we will need to install `curl` in debian-like minions, probably through cloud-init phase.

Comment (jordimassaguerpla) [reply testsuite/features/step_definitions/navigation_steps.rb:1052]: Sorry, I did not realize there was an if ... the issue is that wget is not in Leap Micro, so we do not really need this change here.

Comment (jordimassaguerpla) [reply testsuite/features/step_definitions/navigation_steps.rb:1052]: fixed in https://github.com/uyuni-project/uyuni/pull/8391

**Pull request closed without merging**